esmerald/applications.py

```diff
@@ -2415,37 +2427,9 @@ def build_middleware_stack(self) -> "ASGIApp":
             app = cls(app=app, *args, **kwargs)  # noqa
         return app
 
-    def build_pluggable_stack(self) -> Optional["Esmerald"]:
-        """
-        Validates the pluggable types passed and builds the stack
-        and triggers the plug
-        """
-        if not self.pluggables:
-            return None
-
-        pluggables = {}
-
-        for name, extension in self.pluggables.items():
-            if not isinstance(name, str):
-                raise ImproperlyConfigured("Pluggable names should be in string format.")
-            elif isinstance(extension, Pluggable):
-                pluggables[name] = extension
-                continue
-            elif not is_class_and_subclass(extension, Extension):
-                raise ImproperlyConfigured(
-                    "An extension must subclass from esmerald.pluggables.Extension and added to "
-                    "a Pluggable object"
-                )
-
-        app: "ASGIApp" = self
-        for name, pluggable in pluggables.items():
-            for cls, options in [pluggable]:
-                ext: "Extension" = cls(app=app, **options)
-                ext.extend(**options)
-                self.pluggables[name] = cls
-        return cast("Esmerald", app)
-
-    def add_pluggable(self, name: str, extension: "Extension") -> None:
+    def add_extension(
```

Comment:
We need to keep the add pluggable function and map to add_extension for backwards compatibility. We add a warning for deprecation for the release after the next and then yes this can be removed.